packages/typography/css/fonts.css

```diff
@@ -0,0 +1,42 @@
+@font-face {
```

Comment:
Do you see use for the CSS file already? I think I'd stick to just sass file for now and add CSS only when we actually need it.

If we do need CSS file, I would build it from the sass file into /dist/ folder and not keep it in the version control. Otherwise we need to maintain two separate files and be needlessly mindful about not getting them unsync.

Thoughts?

The sass file could be either in root, in src folder, styles folder, or in sass folder like it is now. No big difference to me. :-) We can also easily change it later.

Reply:
The idea was to avoid a build step for this package, making it a simple folder with static files, with no code to execute.

If the public API of the `fonts.css` stylesheet is only the `.wp-brand-font` class name, and not, for example, any SCSS variables like `$serif` or `$brand-serif`, then we can go the other way around: drop the SCSS file and ship only the CSS one.

Reply:
Right, just felt like sass is pretty convenient to produce that languages-list but I agree CSS is enough here too. :-)

Reply:
If we ship only the SCSS file, and transpiling that into CSS is always the responsibility of the package user, I'm also fine with that. I agree that all real-world consumers of the package will likely be SCSS-capable.